docs/api/hardhat/hardhat-zksync-solc.md

```diff
@@ -52,13 +56,16 @@ Compilers are no longer released as Docker images and its usage is no longer rec
 
 :::
 
-- `version` is a field with the version of the `zksolc` compiler. Compiler versions can be found in [the following repository](https://github.com/matter-labs/zksolc-bin).
+- `version` is the `zksolc` compiler version. Compiler versions can be found in [the following repository](https://github.com/matter-labs/zksolc-bin).
 - `compilerSource` indicates the compiler source and can be either `docker` or `binary` (recommended). If there isn't a compiler binary already installed, the plugin will automatically download it. If `docker` is used, you'd need to run Docker desktop in the background and provide both `dockerImage` and `tag` in the experimental section.
```

Comment:
Should we remove it?

Reply:
I'll keep it. Actually we have no other place in which we mention where to find the compiler versions so this is kinda helpful